pageserver/src/remote_storage.rs

```diff
@@ -280,6 +283,9 @@ fn collect_timeline_files(
         if entry_path.is_file() {
             if entry_path.file_name().and_then(ffi::OsStr::to_str) == Some(METADATA_FILE_NAME) {
                 timeline_metadata_path = Some(entry_path);
+            } else if is_ephemeral_file(entry_path.file_name().unwrap().to_str().unwrap()) {
```

Comment:
> `.to_str().unwrap()`

NIT: could be replaced with `&entry_path.file_name().unwrap().to_string_lossy()` to have one less possible panic